misc-utils/rename.1.adoc

```diff
@@ -49,6 +49,12 @@ include::man-common/help-version.adoc[]
 
 The renaming has no safeguards by default or without any one of the options *--no-overwrite*, *--interactive* or *--no-act*. If the user has permission to rewrite file names, the command will perform the action without any questions. For example, the result can be quite drastic when the command is run as root in the _/lib_ directory. Always make a backup before running the command, unless you truly know what you are doing.
 
+== EDGE CASES
+
+If the _expression_ is empty, then by default _replacement_ will be added to the start of the filename. With *--all*, _replacement_ will be inserted in between every two characters of the filename, as well as at the start and end.
+
+Normally, only the final path component of a filename is updated. But if either _expression_ or _replacement_ contains a _/_, the full path is updated. This can cause a file to be moved between folders. Creating folders, and moving files between filesystems, is not supported. With *--symlink*, the update is always applied to the link's full path.
```

Comment:
I'm not convinced this symlink behavior is intended. Maybe it should be documented as a bug?

Reply:
Good point. I do not see why --symlink behavior should be different. IMHO the same behavior should be applied in all cases to keep the command semantic consistent.

But it is probably better to fix it with a separate patch.